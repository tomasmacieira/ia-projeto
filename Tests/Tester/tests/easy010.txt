ROW	0	4	0	3	4	1	1	2	2	3
COLUMN	1	3	2	2	5	1	0	2	1	3
7
HINT	3	4	T
HINT	1	4	R
HINT	4	4	M
HINT	8	1	R
HINT	8	0	L
HINT	5	0	W
HINT	1	3	M
