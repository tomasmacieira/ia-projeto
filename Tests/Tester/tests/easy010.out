..........
.lmMR.....
..........
.lr.T.....
....M..lmr
W...b.....
.......c..
...c.....c
LR........
....lr...c
